src/ucp/core/ucp_worker.c

```diff
@@ -3102,29 +3102,39 @@ void ucp_worker_keepalive_remove_ep(ucp_ep_h ep)
         ucs_assert(worker->keepalive.iter != &ucp_ep_ext_gen(ep)->ep_list);
 
         if (worker->keepalive.lane_map != 0) {
-            ucs_assert((ucs_get_time() - worker->keepalive.last_round) >=
-                               worker->context->config.ext.keepalive_interval);
+            /* Finish the current keepalive round */
+#if UCS_ENABLE_ASSERT
+            now = ucs_get_time();
```

Comment:
pls move the comment from line 3105 to line 3118, and add a one-line comment what this assert is checking

Reply:
I've moved the assertion to separate function to use it in other branches too